cmdeploy/src/cmdeploy/tests/online/test_1_basic.py

```diff
@@ -209,8 +209,14 @@ def test_expunged(remote, chatmail_config):
 
 
 def test_deployed_state(remote):
-    git_hash = subprocess.check_output(["git", "rev-parse", "HEAD"]).decode()
-    git_diff = subprocess.check_output(["git", "diff"]).decode()
+    try:
+        git_hash = subprocess.check_output(["git", "rev-parse", "HEAD"]).decode()
+    except Exception:
```

Comment:
```suggestion
    except FileNotFoundError:
```

There might be other exceptions - we want to explicitly catch that git isn't installed, right?